frontend/components/dashboard/progress-card.tsx

```diff
@@ -1,20 +1,29 @@
 'use client'
 
 import { Progress } from '@/components/ui/progress'
+import { Complexity } from '@repo/user-types'
 
 interface ProgressCardProps {
-    complexity: string
-    score: string
+    complexity: Complexity
+    completed: number
+    total: number
     progress: number
     indicatorColor: string
     backgroundColor: string
 }
 
-export const ProgressCard = ({ complexity, score, progress, indicatorColor, backgroundColor }: ProgressCardProps) => {
+export const ProgressCard = ({
+    complexity,
+    completed,
+    total,
+    progress,
+    indicatorColor,
+    backgroundColor,
+}: ProgressCardProps) => {
     return (
         <div className="border-solid border-2 border-gray-200 rounded flex flex-col w-2/6 mx-2 p-6">
             <p className="text-medium font-medium">{complexity}</p>
```

Comment:
since youre using an enum now, the complexity is showing in entirely uppercase (can u call the `capitalizeFirstLowerRest` function here, it doesnt look as good.)